src/styles/index.scss

```diff
@@ -1,3 +1,78 @@
+@import './utils/variables';
+
 body {
   margin: 0;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+  color: #000;
+}
+
+.page {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  box-sizing: border-box;
+  width: $field-width;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  padding: $field-pad;
+
+  @each $number, $day in $week {
+    &--start-day-#{$day} > .calendar__day:first-child {
+      margin-left: ($number - 1) * ($cell-size + $cell-gap);
+    }
+  }
+
+  // @for $last-day from 28 through 30 {
+  //   &--month-length-#{$last-day} {
+  //     @for $day from ($last-day + 1) through $days {
+  //       > .calendar__day:nth-child(#{$day}) {
+  //         display: none;
+  //       }
+  //     }
+  //   }
+  // }
+
+  @each $last-day in $possible-month-lengths {
+    &--month-length-#{$last-day} {
+      @if $last-day < $days {
```

Comment:
The variable `$days` is used here but is not defined in this file. Ensure it is imported correctly from `utils/variables` or another file.